src/EnergyPlus/DXCoils.cc

```diff
@@ -15329,11 +15383,40 @@ void CalcTwoSpeedDXCoilStandardRating(EnergyPlusData &state, int const DXCoilNum
                 EIR = 0.0;
             }
 
-            TotCapFlowModFac = CurveValue(state, state.dataDXCoils->DXCoil(DXCoilNum).CCapFTemp2, AirMassFlowRatio);
+            // TotCapFlowModFac = CurveValue(state, state.dataDXCoils->DXCoil(DXCoilNum).CCapFFlow(1), AirMassFlowRatio);
             TotCapTempModFac = CurveValue(state,
                                           state.dataDXCoils->DXCoil(DXCoilNum).CCapFTemp2,
                                           CoolingCoilInletAirWetBulbTempRated,
                                           OutdoorUnitInletAirDryBulbTempPLTestPoint(PartLoadTestPoint));
+            //    Warn user if curve output goes negative
+            if (TotCapTempModFac < 0.0) {
+                if (state.dataDXCoils->DXCoil(DXCoilNum).CCapFTempErrorIndex == 0) {
+                    ShowWarningMessage(
```

Comment:
This and a flow other code blocks add the curve value bound check, so that a negative curve value could be intercepted with warning messages and the values reset to some proper values (mostly zeroes).

Reply:
OK that's fine.